aten/src/ATen/native/cpu/UpSampleKernel.cpp

```diff
@@ -1021,6 +1021,61 @@ struct HelperInterpCubic : public HelperInterpBase {
     );
     return output;
   }
+
+  // taken from
+  // https://github.com/python-pillow/Pillow/blob/6812205f18ca4ef54372e87e1a13ce4a859434df/
+  // src/libImaging/Resample.c#L46-L62
+  template<typename scalar_t>
+  static inline scalar_t aa_filter(scalar_t x) {
+    // https://en.wikipedia.org/wiki/Bicubic_interpolation#Bicubic_convolution_algorithm
+#define a -0.5
```

Comment:
Is there a reason why not to use `scalar_t a = -0.5;` ?

Reply:
No reason, just copy-pasted the code from Pillow to show that here are really doing like them for consistency.